tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/SerilogTests.cs

```diff
@@ -14,40 +14,20 @@ namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
     public class SerilogTests : LogsInjectionTestBase
     {
-        private readonly LogFileTest[] _log200FileTests =
-            {
-                new LogFileTest()
-                {
-                    FileName = "log-textFile.log",
-                    RegexFormat = @"{0}: {1}",
-                    UnTracedLogTypes = UnTracedLogTypes.EmptyProperties,
-                    PropertiesUseSerilogNaming = true
-                },
-                new LogFileTest()
-                {
-                    FileName = "log-jsonFile.log",
-                    RegexFormat = @"""{0}"":{1}",
-                    UnTracedLogTypes = UnTracedLogTypes.None,
-                    PropertiesUseSerilogNaming = true
-                }
-            };
-
-        private readonly LogFileTest[] _logPre200FileTests =
-        {
+        private readonly LogFileTest _txtFile =
             new LogFileTest()
             {
                 FileName = "log-textFile.log",
                 RegexFormat = @"{0}: {1}",
-                TracedLogTypes = TracedLogTypes.NotCorrelated,
                 UnTracedLogTypes = UnTracedLogTypes.EmptyProperties,
```

Comment:
I was expecting `UnTracedLogTypes = UnTracedLogTypes.EnvServiceTracingPropertiesOnly` and it looks like the instrumentation should be doing that. Is there something I'm missing?

Reply:
Ah okay, I looked at the logs and here is are two example log lines:
```
2021-12-09 15:58:14.350 -08:00 [Information] { dd_service: "LogsInjection.Serilog", dd_version: "1.0.0", dd_env: "integration_tests", dd_trace_id: "", dd_span_id: "" } [ExcludeMessage]Entering Datadog scope. 
2021-12-09 15:58:14.423 -08:00 [Information] { dd_service: "LogsInjection.Serilog", dd_version: "1.0.0", dd_env: "integration_tests", dd_trace_id: "4789817333944249620", dd_span_id: "2367892056550244520" } Calling the PluginApplication.Program in a separate AppDomain 
```

This is great and it's exactly what I expected, but we need to update `LogsInjectionTestBase` to better handle this case. I wouldn't consider that blocking for this PR. This same comment applies to the NLog automatic instrumentation as well

Reply:
Ah, good point, thanks, I missed that. I agree, we should add that pattern as an extra option to `LogsInjectionTestBase`

Reply:
This applies to all the automatic instrumentation so I'll implement this in a standalone PR